lib/kube/proxy/auth_test.go

```diff
@@ -280,6 +288,7 @@ current-context: foo
 				cmp.AllowUnexported(kubeDetails{}),
 				cmp.Comparer(func(a, b *transport.Config) bool { return (a == nil) == (b == nil) }),
 				cmp.Comparer(func(a, b *kubernetes.Clientset) bool { return (a == nil) == (b == nil) }),
+				cmp.Comparer(func(a, b *rest.Config) bool { return (a == nil) == (b == nil) }),
```

Comment:
Could we use cmpopts.IgnoreFields (or something to that effect)? It makes the semantics clearer.

https://pkg.go.dev/github.com/google/go-cmp@v0.5.9/cmp/cmpopts#IgnoreFields